modules/claims_api/spec/concerns/claims_api/v2/claims_requests/claim_validation_spec.rb

```diff
@@ -0,0 +1,150 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'bd/bd'
+
+class FakeController
+  include ClaimsApi::V2::ClaimsRequests::ClaimValidation
+
+  def target_veteran
+    OpenStruct.new(
+      icn: '1013062086V794840',
+      first_name: 'abraham',
+      last_name: 'lincoln',
+      loa: { current: 3, highest: 3 },
+      ssn: '796111863',
+      edipi: '8040545646',
+      participant_id: '600061742',
+      mpi: OpenStruct.new(
+        icn: '1013062086V794840',
+        profile: OpenStruct.new(ssn: '796111863')
+      )
+    )
+  end
+end
+
+describe ClaimsApi::V2::ClaimsRequests::ClaimValidation do
+  let(:target_veteran) do
+    OpenStruct.new(
+      icn: '1013062086V794840',
+      first_name: 'abraham',
+      last_name: 'lincoln',
+      loa: { current: 3, highest: 3 },
+      ssn: '796111863',
+      edipi: '8040545646',
+      participant_id: '600061742',
+      mpi: OpenStruct.new(
+        icn: '1013062086V794840',
+        profile: OpenStruct.new(ssn: '796111863')
+      )
+    )
+  end
+
+  let(:dependent_target_veteran) do
+    OpenStruct.new(
+      icn: '1012861229V078999',
+      first_name: 'Janet',
+      last_name: 'Moore',
+      loa: { current: 3, highest: 3 },
+      ssn: '796127677',
+      edipi: '8040545646',
+      participant_id: '600987654',
+      mpi: OpenStruct.new(
+        icn: '1012861229V078999',
+        profile: OpenStruct.new(ssn: '796127677')
+      )
+    )
+  end
+
+  let(:controller) { FakeController.new }
+
+  let(:vet_request_icn) { target_veteran.icn }
+  let(:claimant_request_icn) { target_veteran.icn }
+  let(:bgs_claim_for_target_veteran) do
+    { benefit_claim_details_dto: { ptcpnt_vet_id: '600061742', ptcpnt_clmant_id: '600987654' } }
+  end
+  let(:bgs_claim_for_dependent) do
+    { benefit_claim_details_dto: { ptcpnt_vet_id: '600061742', ptcpnt_clmant_id: '600987654' } }
+  end
+  let(:inaccessable_bgs_claim) do
+    { benefit_claim_details_dto: { ptcpnt_vet_id: '999999999', ptcpnt_clmant_id: '888888888' } }
+  end
+
+  context 'when the claims controller calls the claim_validation module' do
```

Comment:
Do we need this outer context?

Reply:
I wanted to keep it inline with the other files like this going in, but it certainly would be fine without it.

Reply:
It doesn't hurt - just might be misleading if we (hopefully) reuse this concern elsewhere in the future.

Reply:
Ya makes sense, got it removed